### PR TITLE
test mochitest runner

### PR DESCRIPTION
**Pull request by jlongster**
Triggering a circle run to see if docker will work...

### CODE REVIEW

**Comment by jlongster** [comment]
Holy shit, got it to work!

File: circle.yml [hunk 1]
```diff
@@ -9,6 +11,7 @@ test:
   override:
     - mkdir -p $CIRCLE_TEST_REPORTS/mocha
     - node public/js/test/node-unit-tests.js --ci
+    - docker run -it -e "DISPLAY=unix$DISPLAY" -v "/tmp/.X11-unix:/tmp/.X11-unix:rw" --ipc host jlongster/mochitest-runner /bin/bash -c "export SHELL=/bin/bash; cd firefox && ./mach mochitest --subsuite devtools devtools/client/debugger/test/mochitest/browser_dbg_breakpoints-pane.js"
```
Comment (jasonLaster) [inline circle.yml]: can we move this to a script:

`docker run -it ./bin/run-mochitest`

**Comment by jasonLaster** [comment]
Awesome!

**Comment by jlongster** [comment]
Closing in favor of #630

**Pull request closed without merging**